docs/README.md

```diff
@@ -1,56 +1,171 @@
-<!-- 📢 Use this project, [contribute](https://github.com/vtex-apps/product-gifts) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). -->
+📢 Use this project, [contribute](https://github.com/vtex-apps/product-gifts) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
 
-📢 **This project is a work-in-progress, this app is not read for use yet.**
+# Product Gifts
 
-# [WIP] Product Gifts
+The Product Gifts app provides blocks responsible for displaying, in the Product Description block, all gifts available for a given product.
 
-Under the block's name, you should explain the topic, giving a **brief description** of the **block's functionality** in a store.
+![product-gifts-in-pdp](https://user-images.githubusercontent.com/27777263/75767692-f68d9800-5d21-11ea-87b9-aa4544f03698.png)
 
-Next, **add media** (either an image of a GIF) with the rendered block, so that users can better understand how the block works in practice.
-
-![Media Placeholder](https://user-images.githubusercontent.com/52087100/71204177-42ca4f80-227e-11ea-89e6-e92e65370c69.png)
+:information_source: *A product's gift is configured in a [Buy&Win promotion](https://help.vtex.com/tutorial/buy-and-win--tutorials_322)
```

Comment:
:information_source: *A product's gift is configured in a [Buy&Win promotion](https://help.vtex.com/tutorial/buy-and-win--tutorials_322)*.

Reply:
it was missing a * at the end of the sentence